src/Component/Router.purs

```diff
@@ -69,10 +72,10 @@ component
   => ManageArticle m
   => ManageComment m
   => ManageTag m
-  => H.Component HH.HTML Query Unit Void m
+  => H.Component HH.HTML Query Input Void m
 component =
   H.parentComponent
-    { initialState: \_ -> { route: Home } 
+    { initialState: \i -> { route: fromMaybe Home i } 
```

Comment:
I'm not always great about this, but the longer I write PureScript the more I favor readable variable names. I think `initialRoute` was a good choice to name this argument in the `Main` module. What do you think of renaming the variable here to be the same so it's crystal clear at a glance what's going on?

Reply:
That sounds good. Just force-pushed to my fork with your suggestion to keep the history a bit cleaner.